bauble/src/value/error.rs

```diff
@@ -0,0 +1,689 @@
+use std::{borrow::Cow, collections::HashMap};
```

Comment:
same here?

Reply:
Basically not changed, except for the addition of one error, `VariantErrors`